cg/meta/workflow/analysis.py

```diff
@@ -423,3 +427,39 @@ def parse_analysis(
         """Parses output analysis files"""
 
         raise NotImplementedError
+
+    def clean_analyses(self, case_id: str) -> None:
+        """Add a cleaned at date for all analyses related to a case."""
+        analyses: list = self.status_db.family(case_id).analyses
+        LOG.info(f"Adding a cleaned at date for case {case_id}")
+        for analysis_obj in analyses:
+            analysis_obj.cleaned_at = analysis_obj.cleaned_at or dt.datetime.now()
+            self.status_db.commit()
+
+    def clean_run_dir(self, case_id: str, yes: bool, dry_run: bool = False):
```

Comment:
Unit test

Reply:
@henrikstranneheim There some larger unit tests for a balsamic clean in tests/cli/clean/test_balsamic_clean.py. Are these sufficient or should I create a similar one for MicroSALT?

Reply:
It is great if all functions has their own unit tests (if possible)